backend/cloud_inquisitor/schema/base.py

```diff
@@ -74,155 +73,155 @@ def to_json(self):
-            return {
-                'accountId': self.account_id,
-                'accountName': self.account_name,
-                'accountNumber': self.account_number,
-                'accountType': self.account_type,
-                'contacts': self.contacts,
-                'adGroupBase': self.ad_group_base,
-                'enabled': True if self.enabled == 1 else False,
-                'requiredRoles': self.required_roles
-            }
-        else:
-            return {
-                'accountId': self.account_id,
-                'accountName': self.account_name,
-                'contacts': self.contacts
-            }
-
-    def user_has_access(self, user):
-        """Check if a user has access to view information for the account
-
-        Args:
-            user (:obj:`User`): User object to check
-
-        Returns:
-            True if user has access to the account, else false
-        """
-        if ROLE_ADMIN in user.roles:
-            return True
-
-        # Non-admin users should only see active accounts
-        if self.enabled:
-            if not self.required_roles or len(self.required_roles) == 0:
-                return True
-
-            for role in self.required_roles:
-                if role in user.roles:
-                    return True
-
-        return False
-
-    def delete(self, auto_commit=True):
-        """Delete the account and all of its resources from the database
-
-        Args:
-             auto_commit (`bool`): Automatically commit the change to the database. Default: `True`
-
-        Returns:
-            `None`
-        """
-        log.info('Deleting account {}'.format(self.account_name))
-        db.session.delete(self)
-        if auto_commit:
-            db.session.commit()
-
-    def get_contacts(self):
-        return [NotificationContact(contact['type'], contact['value']) for contact in self.contacts]
-
-    @classmethod
-    def get(cls, account):
-        """Return an Account object by name
-
-        Args:
-            account (str): Name of the account to lookup
-
-        Returns:
-            Account object or None if not found
-        """
-        if isinstance(account, str):
-            return getattr(db, cls.__name__).find_one(
-                Account.account_name == account
-            )
-
-        elif isinstance(account, int):
-            return getattr(db, cls.__name__).find_one(
-                Account.account_number == account
-            )
-
-        elif isinstance(account, cls):
-            return account
-
-        else:
-            return None
-
-    @classmethod
-    def get_by_id(cls, account_id):
-        """Returns an Account object based on the account_id
-
-        Args:
-            account_id (int): ID of the account to fetch
-
-        Returns:
-            :obj:`Account`
-        """
-        return getattr(db, cls.__name__).find_one({
-            'account_id': account_id
-        })
+# class OldAccount(Model, BaseModelMixin):
```

Comment:
Can you add a comment above this indicating why all this code is commented out?

Reply:
Should be deleted